easybuild/easyconfigs/o/ordereddict/ordereddict-1.1-goolf-1.4.10-Python-2.7.3.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = "PythonPackage"
+
+name = 'ordereddict'
+version = '1.1'
+
+homepage = 'https://pypi.python.org/pypi/ordereddict'
+description = """Drop-in substitute for Py2.7's new collections.OrderedDict. 
+The recipe has big-oh performance that matches regular dictionaries (amortized 
+O(1) insertion/deletion/lookup and O(n) iteration/repr/copy/equality_testing)."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://pypi.python.org/packages/source/o/%s/' % name.lower()]
+sources = [SOURCELOWER_TAR_GZ]
+
+python = "Python"
+pythonversion = '2.7.3'
+pythonshortversion = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [(python, pythonversion)]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['.'],
```

Comment:
or, just drop `sanity_check_paths`; for stuff installed via `PythonPackage`, EasyBuild already does an `import` test, which should be good enough?